scripts/monitoring/delete-stuck-projects.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This is a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1367432
+
+# Look for any projects found during the last run of this script
+if [ -f /tmp/stuck_projects ]; then
+  projects_to_delete="$(cat /tmp/stuck_projects)"
+
+  # If the project still exists, is empty, and contains nothing, delete it.
+  for project in ${projects_to_delete}; do
+    oc get project ${project} &> /dev/null
+
+    if [ $? == 0 ] && [ -z "$(oc get all -n ${project})" ]; then
```

Comment:
Instead of getting the project in the line above and then getting all resources within the project here in this condition, could we not simply just get all resources in the project (just that one call)? We can check if the exit code is 0 and if the result is empty, right?